samples/bluetooth/direct_test_mode/conf/remote_shell_nrf53.overlay

```diff
@@ -0,0 +1,13 @@
+/ {
+	chosen {
```

Comment:
Why we need this additional file?

Reply:
It allows using the remote_shell as transport instead of remote_hci.